webui/react/src/components/Navigation.tsx

```diff
@@ -167,7 +167,7 @@ const Navigation: React.FC = () => {
           <section className={css.bottom}>
             <NavigationItem icon="logs" label="Master Logs" path="/det/logs" popout />
             <NavigationItem icon="docs" label="Docs" noProxy path="/docs" popout />
-            <NavigationItem icon="cloud" label="API" noProxy path="/swagger-ui" popout />
+            <NavigationItem icon="cloud" label="API (Beta)" noProxy path="/swagger-ui" popout />
```

Comment:
question: Do we want to use small text for this (or sub-/superscript)?

Reply:
yeah, that'd be good. it means we can't just use a plain label but the change is simple. I made the change

Reply:
Nah, just go with what you had!

Reply:
nit suggestion: Not sure how long the `Beta` tag is going to stick around. This is probably way too much work than needed but could also convert `label` prop for `NavigationItem` to be `React.ReactNode` and convert `(Beta)` to be `<Badge>Beta</Badge>`. For it to show up properly on the navigation bar, it would probably need additional styling too. Feel free to ignore this.

Reply:
I'll stick with the simple change to get the main point in then we can set up another PR to make it look nicer.

Reply:
> could also convert label prop for NavigationItem to be React.ReactNode and convert (Beta) to be <Badge>Beta</Badge>

yeah that's a good idea too. I think we'd need a separate prop for augmenting the title attribute in this case